app/assets/javascripts/discourse/app/components/d-lightbox/carousel.hbs

```diff
@@ -1,10 +1,10 @@
 <div class="d-lightbox__carousel">
   {{#if @shouldDisplayCarouselArrows}}
-    <DLightbox::Button
-      @class="d-lightbox__carousel-previous-button"
-      @name="previous"
+    <D::Button
```

Comment:
Is github tricking me? it seems like it shows "D::Button" and should be "DButton"

Reply:
No worries, and yes I guess both can work